src/native/libs/configure.cmake

```diff
@@ -226,6 +226,11 @@ check_symbol_exists(
     sys/ioctl.h
     HAVE_IOCTL)
 
+check_symbol_exists(
+    getrandom
+    sys/random.h
+    HAVE_GETRANDOM)
+
```

Comment:
Can we move this check to the minipal CMake files? That way it will for sure show up for all usages of the minipal.

Reply:
@jkoritzinsky You're going to have to help me by being a little more specific. It's also in `src/native/minipal/configure.cmake`, which is what I think what you are referring to. Are you saying it should only be needed there and remove it from here?

Reply:
Yes it should only be needed in the minipal. It shouldn't be needed in the libs at all.